src/main/java/com/github/firmwehr/gentle/source/Source.java

```diff
@@ -219,8 +242,12 @@ private void formatLastLineOfMultilineMessage(
 		StringBuilder codeLine = new StringBuilder();
 		StringBuilder noteLine = new StringBuilder();
 
-		codeLine.append(("%" + lineNrWidth + "d").formatted(endLineNr)).append(" | | ");
-		noteLine.append(" ".repeat(lineNrWidth)).append(" | `-");
+		codeLine.append(LINE_COLOR)
+			.append(("%" + lineNrWidth + "d").formatted(endLineNr))
+			.append(" |")
+			.append(ERROR_COLOR)
```

Comment:
This `ERROR_COLOR` is not reset if the span doesn't extend to the end of the line